WEEK 08/js/app.js

```diff
@@ -0,0 +1,92 @@
+var obj = {
+	data: {
+		user:'Yang',
+		message: 'Woot my first message!',
+	}
+}
+
+var url = 'https://fewd20.herokuapp.com/chat';
+
+var usedNames= [];
+
+$(document).ready(function(){
+	$.get(url, function(data, status){
+
+		for(var i=0; i< data.length; i++){
+
+			if (usedNames.indexOf(data[i].name) === -1){
+				usedNames.push(data[i].name);
+				$('#printname').append(data[i].user);
+				$('#printmessage').append(data[i].message);
+			}
+		}
+	}).fail(function(){
+
+	});
+})
+
+$('#send').click(function(){
+	$.post(url, obj, function(data, status){
+		console.log(data)
```

Comment:
Do a GET here to get the message again and show it. Pretty much what you have above.